### PR TITLE
feat: add metamask signature in certificate.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#535

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +37,14 @@
...
         "stream-browserify": "^3.0.0",
+        "stream-http": "^3.2.0",
         "text-encoding": "^0.7.0",
+        "url": "^0.11.0",
         "uuid": "^8.3.2",
         "web-vitals": "^2.1.4"
     },
     "scripts": {
-        "start": "craco start",
+        "start": "GENERATE_SOURCEMAP=false craco start",
```
Comment (gdethier) [inline package.json:47]: What's the purpose of this?

File: src/certificate/Certificate.test.tsx [hunk 2]
```diff
@@ -16,9 +16,10 @@ jest.mock("@logion/node-api/dist/LogionLoc");
 jest.mock("../common/api");
 jest.mock("../common/hash");
 jest.mock("../common/Model");
+jest.mock('@logion/extension');
```
Comment (gdethier) [inline src/certificate/Certificate.test.tsx:19]: This is implicit. You may safely remove this line.

**Review by gdethier — APPROVED** [review]
Good job! I left a few remarks. Have a nice Weekend!

File: package.json (hunk 1 above)
Comment (benoitdevos) [reply package.json:47]: > What's the purpose of this?

Avoid a big bunch of warnings (`Failed to parse source map`...)

File: src/certificate/Certificate.test.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/certificate/Certificate.test.tsx:19]: OK, thanks !. Will be done in the next PR.

**Pull request merged**